### ISSUE TITLE
Where to get the information to fill in the sample information file structure?

### ISSUE THREAD

**Issue by matsojr22** [ZadorLaboratory/mapseq-processing#3]
> worksheet entitled "Sample Information", and that there be columns titled: "RT primers for MAPseq", "Brain", "Region", "Our Tube #", "Site information"

The Mseq### file returned by the core contains some of this info but not all. Is there a way to get the correct values for these columns?

**Comment by jhover** [comment]
Good point. That should become a standard thing returned from the core, as the mappings (rtprimers -> region) are required (and can't be derived). We'll send it to you.
